just readme.md

```diff
@@ -0,0 +1,8 @@
+In this file you can find useful information about this project.
+
+* This project has a home page called index.html where you can see a greeting message "Hello Microverse" in blue color.
+
+* There is a styles.css file which contains style properties for the H1 header element.
+
+* There is a file named .gitignore which ignores unnecessary files from this project.
+
```

Comment:
- [ ] Your  README file should be descriptive as required for this project, Kindly make your readme descriptive and more professional, you can use AKA: Microverse README [template](https://github.com/microverseinc/readme-template) to make your README file professional. :wink:

Reply:
@Alfredbis29 I have used Microverse Readme template in my Readme file. Since it's a beginner-level project, therefore, description of the project is not very lengthy. I have given as much information as required for this small project.